### DISCUSSION TITLE
Config not found

### DISCUSSION THREAD

**Discussion by simondsmason** [QuantumEntangledAndy/neolink#224] [Q&A]
I had Neolink up and running then cache drive crashed and I am rebuilding everything.

I can't see to get it to find the config file - and my lack of understanding of the inner workings of docker and Unraid is failing me.  My settings are below.

neolink.toml is stored in mnt/user/appdata/neolink.

The error I am getting is:  /entrypoint.sh: 9: --config=/config/neolink.toml: not found

I have tried a few variations of this.  Appreciate any help.  Thanks.

![image](https://github.com/QuantumEntangledAndy/neolink/assets/6063262/b7d696b2-c990-4c25-b620-7f7f7e136b65)

![image](https://github.com/QuantumEntangledAndy/neolink/assets/6063262/cbd3e90d-2e23-4925-8d9f-c87d8bc6cb08)

**Comment by simondsmason** [comment]
Figured it out with some help from Reddit.  Had to add a path in to point to the config file in appdata.  Got it loading right now but still have other issues to work through separate from this.

**Reply by QuantumEntangledAndy** [reply]
I see glad you figured it out. I don't use a gui or webfront end for my own containers so I am never sure about how to address these

**Reply by simondsmason** [reply]
For those that may have the same question on unraid - I created a new path with the following settings:

Name: config
Host path: /mnt/user/appdata/neolink/neolink.toml <- This may vary depending on where you store the config file

Save that and then it will find the config file.

As mentioned, everything is loading for me but I can't read the RTSP so not sure that I have it sorted yet.  But that is not an issue with finding the config file so will post separately about that.
